docs/developer-guide/nodejs-api.md

```diff
@@ -355,7 +355,9 @@ The `LintResult` value is the information of the linting result of each file. Th
 * `fixableWarningCount` (`number`)<br>
   The number of warnings that can be fixed automatically by the `fix` constructor option.
 * `errorCount` (`number`)<br>
-  The number of errors. This includes fixable errors.
+  The number of errors. This includes fixable errors and fatal errors.
+  `fatalErrorCount` (`number`)<br>
```

Comment:
```suggestion
* `fatalErrorCount` (`number`)<br>
```

Missing `*` so it doesn't appear as a list item ([preview link](https://github.com/eslint/eslint/blob/f9ff856f00fc76ea9d041d5f376018d8a9be714f/docs/developer-guide/nodejs-api.md#-lintresult-type))